.github/workflows/release.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           test_run=false
           ENTIRE_TAG_RE="^(([[:alpha:]\-]+)-)?(.*)$"
           VERSION_TAG_RE="^v[\.]?([[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+(-[[:alpha:][:digit:]\.]+)*)$"
-          TAG="${{ github.event.release.tag_name }}"
+          TAG="${FULL_TAG}"
```

Comment:
nit: not sure of all the syntactic differences, but this is what chatGPT suggested and I was able to test with successfully yesterday
```suggestion
          TAG="${{env.FULL_TAG}}"
```